packages/website/sidebars/sidebar.rules.js

```diff
@@ -43,10 +43,7 @@ const paths = globby
   });
 
 function createCategory(label, rules, additionalItems = []) {
-  const collapsed = !additionalItems.length;
   return {
-    collapsed,
-    collapsible: collapsed,
```

Comment:
Would you mind moving the sidebar changes for #5554 to another PR? That way we can get them in sooner - and it's easier to keep track when it's one-issue-per-PR.